doc/articles/figma-plugin/designers/simple-login-page.md

```diff
@@ -0,0 +1,193 @@
+
+3. **Top App Bar:** The sticky top bar that contains the commands and page title. If this layer is invisible, this component is not relevant to the actual page, and it could be deleted from the layers.
+
+4. **Page:** Contain all the Content. Scrollable layers and the Top App Bar component.
+
+5. **Content.Scrollable:** Contain all the scrollable content.
+
+6. **Section Placeholder (2):** We added 2 section placeholder to help you insert content faster into your design
+
+7. **Navigation Bar:** The sticky bottom navigation bar. If this layer is invisible, this component is not relevant to the actual page, and it could be deleted from the layers.
+
+8. **App bar:** iOS or Android top app bar. Not part of the actual app, but it's only for presentation purposes.
+
+## Step 2: Creating a form
+
+![Add text box](images/03_add-text-box.png)
+### How to Add a Text Box?
+1. **Drag the Text Box** from the Text components library (left side of your screen) into your Figma project.
+
+2. **Select the Text Box** by clicking on it. You should see a purple outline around the component.
+
+3. **Customize the look and content** of the Text Box with the options you need from the variants control panel (right side of your screen).
+
+4. **Adjust the width of the Text Box** by changing the value from "fixed width" to "fill container" in the Resizing panel (right side of your screen)
+
+5. **Edit the text of the Text Box.** To do so, you could click on the label until the text becomes highlighted or select the label layer from the layers panel (left side of the screen). Once selected, use your keyboard to enter new text.
+
+6. **Make the Assistive label invisible** if you are not using it.
+
+![Change TextBox icon](images/04_change-textbox-icon.png)
+### How to Change the Icon of a Text Box?
+1. **Select the icon** from a Text Box by clicking directly on the icon multiple times until the icon becomes highlighted (1b) or by selecting the Icon layer from the layer panel (left side of your screen - 1a).
+
+2. **Choose a new icon** from the drop-down menu in the components control panel (right side of the screen).
+
+![Layers of TextBox](images/05_textbox-layers.png)
+### What Are the Layers in a Text Box?
+1. **Template:** Link to the main template
+
+2. **Assistive text:** Text displayed under the text box
+
+3. **Line:** Horizontal line under the text box (only in filled type text box)
+
+4. **Input:** Group containing the Suffix, Icons, and Input text
+
+5. **TrailingIcon:** Icon displayed on the right
+
+6. **Content:** Group containing the Input text and Label
+
+7. **Input text:** Group containing the Cursor, Text, and Prefix
+
+8. **Cursor:** Vertical bar on the right of the text (used only for illustrative purposes)
+
+9. **Text:** Text input by the user
+
+10. **Prefix:** Text displayed on the left
+
+11. **Label:** Name of the text box
+
+12. **LeadingIcon:** Icon displayed on the left
```

Comment:
it can be addressed in a later PR, but I feel like the material here would make more sense in a separate explainer/reference-oriented doc, rather than in the middle of a step-by-step how-to.